go.mod

```diff
@@ -0,0 +1,37 @@
+module github.com/openshift/osde2e
+
+go 1.12
```

Comment:
I'm guessing this would need to change if we moved to the most recent go version.